go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/elastic/package-registry
 
-go 1.22.1
+go 1.22.8
```

Comment:
Updated also to the latest Golang 1.22 version.

Not sure if this is required, since the version defined in the toolchain is going to be downloaded and used.

WDYT ?

Reply:
I would keep it as 1.22.1 if possible. This will be the minimum version required for projects importing this.

Reply:
Ok! Reverting it.